core/src/test/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoEntityProviderSpec.scala

```diff
@@ -212,6 +215,58 @@ class DataRepoEntityProviderSpec extends AsyncFlatSpec with DataRepoEntityProvid
     }
   }
 
+  val table2Result = {
```

Comment:
@andy7i I think this looks fine in general. I would make `table2Result` a function and parameterize table2RowCount. Probably call the function something like `createTestTableResult`